packages/collabswarm/package.json

```diff
@@ -23,9 +23,9 @@
     "jest": "^27.0.4",
     "ts-jest": "^27.0.3",
     "ts-node": "^10.1.0",
-    "typedoc": "^0.21.0",
+    "typedoc": "<0.21.0",
     "typedoc-neo-theme": "^1.1.1",
-    "typescript": "^4.3.5"
+    "typescript": "<4.3"
```

Comment:
What's the reason for this (and the change above it)?

Reply:
Fix build error, see issue in commit msg.